wolfCLU/clu_src/x509/clu_parse.c

```diff
@@ -76,19 +82,54 @@ int wolfCLU_parseFile(char* infile, int inform, char* outfile, int outform,
     /* MALLOC buffer for the certificate to be processed */
     inBuf = (byte*) XMALLOC(MAX_CERT_SIZE, HEAP_HINT,
                                                    DYNAMIC_TYPE_TMP_BUFFER);
+
     if (inBuf == NULL) return MEMORY_E;
     XMEMSET(inBuf, 0, MAX_CERT_SIZE);
 
     instream    = fopen(infile, "rb");
-    outstream   = fopen(outfile, "wb");
-
+    if (XSTRNCMP(outfile, "stdout", 6) == 0) {
+        outstream = stdout;
+    }
+    else {
+        outstream  = fopen(outfile, "wb");
+    }
+    
 /*----------------------------------------------------------------------------*/
 /* read in der, output der */
 /*----------------------------------------------------------------------------*/
     if ( (inform & outform) == 1) {
         printf("in parse: in = der, out = der\n");
     }
 /*----------------------------------------------------------------------------*/
+/* read in pem, output pem formatted human-readable-text */
+/*----------------------------------------------------------------------------*/    
+    else if ( inform == PEM && outform == TEXT ) {
+        X509* x509;
```

Comment:
Though this will work, we should use wolfSSL API/types in wolfCLU instead of relying on macros from wolfssl/openssl/*.h files. i.e. WOLFSSL_X509* x509, and WOLFSSL_BIO*